pom.xml

```diff
@@ -43,6 +50,49 @@
         </pluginRepository>
     </pluginRepositories>
 
+    <properties>
+        <workflow.version>1.8</workflow.version>
+    </properties>
+    <dependencies>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins.workflow</groupId>
+            <artifactId>workflow-job</artifactId>
+            <version>${workflow.version}</version>
+            <scope>test</scope>
```

Comment:
Since there are potential conflicts between Workflow runtime extensions and other stuff, I finally agree that we should minimize the dependency scope to avoid false-positives => no changes needed